packages/vue-searchbox/examples/with-vue-table/src/App.vue

```diff
@@ -82,7 +83,10 @@
                   {{ results.time }}ms
                 </p>
                 <div v-if="!loading && !error">
-                  <v-client-table :data="results.data" :columns="columns" :options="options"></v-client-table>
+                  <v-client-table
```

Comment:
@mohdashraf010897 Is there any reason to remove `options` property?

Reply:
yes, the options weren't defined.